### ISSUE TITLE
js/ directory missing from v4 alpha 3 on npm

### ISSUE THREAD

**Issue by cyChop** [twbs/bootstrap#20385]
[The doc](https://v4-alpha.getbootstrap.com/getting-started/download/#npm) specifies that plugins should be individually available under `js`, but the folder is unavailable:

```shell
$ ll node_modules/bootstrap/
total 53
-rw-r--r-- 1 cyChop 197609   425 Jul 28 01:07 CHANGELOG.md
drwxr-xr-x 1 cyChop 197609     0 Jul 28 01:07 dist/
drwxr-xr-x 1 cyChop 197609     0 Jul 28 01:07 grunt/
-rw-r--r-- 1 cyChop 197609 12864 Jul 28 01:07 Gruntfile.js
-rw-r--r-- 1 cyChop 197609  1131 Jul 28 01:07 LICENSE
-rw-r--r-- 1 cyChop 197609  4690 Jul 28 01:07 package.json
-rw-r--r-- 1 cyChop 197609  7636 Jul 28 01:07 README.md
drwxr-xr-x 1 cyChop 197609     0 Jul 28 01:07 scss/
```

In 4.0.0-alpha.2, the plugins were available under `js/umd`.

(Windows 10, npm 3.6.0)

Thanks for all the great work.

**Comment by cvrebert** [comment]
Possibly related to https://github.com/npm/npm/issues/5082

**Comment by cvrebert** [comment]
Ah, probably the glob in https://github.com/twbs/bootstrap/blob/c7de6107a86ebe65a6f5d5b88884256e5094ca92/package.json#L84 should have been recursive.

**Comment by gilbarbara** [comment]
@mdo 
alpha.4? like 6 months away? :(

**Comment by mdo** [comment]
@gilbarbara No idea when Alpha 4 will drop. If you or anyone else feel they can tackle some of these issues, pull requests to help get us closer to shipping are totally welcomed though!

**Comment by mdo** [comment]
Fix coming in #20418.

**Comment by mdo** [comment]
Merged #20418.
